.github/workflows/ci.yml

```diff
@@ -720,6 +745,15 @@ jobs:
           ABFS_CONTAINER: ${{ secrets.AZURE_ABFS_CONTAINER }}
           ABFS_ACCOUNT: ${{ secrets.AZURE_ABFS_ACCOUNT }}
           ABFS_ACCESS_KEY: ${{ secrets.AZURE_ABFS_ACCESSKEY }}
+          S3_BUCKET: trino-ci-test
+          AWS_REGION: us-east-2
+          TRINO_AWS_ACCESS_KEY_ID: ${{ secrets.TRINO_AWS_ACCESS_KEY_ID }}
+          TRINO_AWS_SECRET_ACCESS_KEY: ${{ secrets.TRINO_AWS_ACCESS_KEY_ID }}
```

Comment:
If you take the `TRINO_` prefix off aws clients will pick them up automatically

Reply:
My intention was to avoid colliding with AWS credentials used by default for the CI runner for other product tests.

Reply:
Do we have AWS credentials used by default for the CI?

Reply:
We still may want to configure env for the containers only, and not the PTL, but it seems like a redundant complication to me.

Reply:
I was under the impression that there are s3 / glue related product tests, but now after looking after any product test environments that have these needs, I didn't find any.
I can change the name of the variables, but they still need to be passed to the corresponding containers.

Reply:
> I was under the impression that there are s3 / glue related product tests,

They should exist, but they don't yet https://github.com/trinodb/trino/issues/5426

Once we have them, we will be invoked explicitly in ci.yml (my assumption)
and then they can set same env vars (with potentially different values)

we would not have ability to define one suite that exercises Hive+Glue and Delta+Databricks, but i don't know whether it will be a loss actually.

> I can change the name of the variables, but they still need to be passed to the corresponding containers.

of course